docs/src/modules/components/withRoot.js

```diff
@@ -7,6 +7,9 @@ import AppWrapper from 'docs/src/modules/components/AppWrapper';
 import initRedux from 'docs/src/modules/redux/initRedux';
 import findPages from /* preval */ 'docs/src/modules/utils/findPages';
 import { loadCSS } from 'fg-loadcss/src/loadCSS';
+import acceptLanguage from 'accept-language';
+
+acceptLanguage.languages(['en', 'zh']);
```

Comment:
I'd suggest we constrain this to `en` for now, get the CrowdIn `l10n` branch going to accept ,Chinese translations, and once we have critical mass, add `zh` back. We could then add a "request for translation" (with a link to CrowdIn) at the top of any `*-zh.md` that are still in English.

Reply:
Should we have a menu to switch better the different languages? I think that shipping the *en* **and** *zh* languages with an edit button would help a lot engaging the Chinese community in the translation.

Reply:
Possibly, but offering an alternative translation when there is none is a false promise, which ever way it's enabled. Better to start with the users who have offered to translate, and a notification based call-to-action first, then make it live when there is some substance, and proof of future intent for others to get behind.